website/docs/reference/widgets/chart.md

```diff
@@ -94,6 +94,7 @@ Let’s use a line chart to visualize the data.
 As you can see in the video, we use the `Chart Series` property head to supply the data and details related to identifying the data points.
 
 * `Series Title` - the name of the series. In the example above, it is _**Total Bugs**_.
+* `Series Color` - a color picker input to select the color of the series (this property is not available for pie chart yet).
```

Comment:
In the video, the series color property is not visible. Can we instead add this in the Widget Properties table for now? https://appsmith-docs-git-18229-add-series-color-to-5ab409-get-appsmith.vercel.app/reference/widgets/chart#widget-properties

The docs team will add a bit of explanation for the property and a separate video later on when you mark this PR with label as 'Ready for docs' team